packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -20,13 +21,14 @@ let sdkFolderPath: string;
 let sdkPackagePath: string;
 const execAppOptions = getDefaultExecSyncOptions();
 const packageManager = getPackageManager();
+const describeSkipIfYarn1 = isYarn1() ? describe.skip : describe;
 
-describe('Create new sdk command', () => {
+describeSkipIfYarn1('Create new sdk command', () => {
```

Comment:
Why do you need to skip this test in case of `yarn 1`?
Is it because of the config? In that case it would maybe make sense to fix it instead

Reply:
we are on it. The goal of the PR will be to have yarn 1 IT tests